src/js/reducers/documentReducer.js

```diff
@@ -47,11 +47,30 @@ function documentReducer(state = initialState, action) {
         documentsError: undefined,
         loadingDocument: true,
       };
-    // case types.DELETE_DOCUMENT_SUCCESS: {
-    //   const newState = Object.assign([], state);
-    //   console.log(newState);
-    //   return newState;
-    // }
+    case types.DELETE_DOCUMENT_SUCCESS: {
+      const newDocs = state.documents.filter(document => document._id !== action.id);
+      return {
+        ...state,
+        documents: [
+          ...newDocs,
+        ],
+        deletingDocument: false,
```

Comment:
I would use just `loading: true | false`

Reply:
should loading: will same variable for all loading actions, like document list or document details, because earlier we talk about this, and we decided that will be better if we have ability to show user  what is currently loading. So i guessed same approach to delete actions

Reply:
ok, agree